### PR TITLE
Add base fee to glossary

### PR DESCRIPTION
**Pull request by zainabb12345**
- Added "Base fee" in the glossary section

## Description

- Provided term's definition and link for better-detailed understanding

### CODE REVIEW

File: src/content/glossary/index.md [hunk 1]
```diff
@@ -59,6 +59,14 @@ A validator vote for a [Beacon Chain](#beacon-chain) or [shard](#shard) [block](
 
 ## B {#section-b}
 
+### Base Fee {#base-fee}
+
+Every Block has a reserve price called the "base fee". It is the minimum fee a user pays to include a transaction in the next block. The base fee helps in making the transaction more predictable.
```
Comment (minimalsm) [inline src/content/glossary/index.md:64]: - Block shouldn't be capitalised mid-sentence
- We should link to block and gas definitions in the glossary
- I think we should drop the last sentence, we should only define base fee in the glossary, not explain why it was introduced.

```suggestion
Every [block](#block) has a reserve price known as the 'base fee'. It is the minimum [gas](#gas) fee a user must pay to include a transaction in the next block.
```

File: src/content/glossary/index.md [hunk 2]
```diff
@@ -59,6 +59,14 @@ A validator vote for a [Beacon Chain](#beacon-chain) or [shard](#shard) [block](
...
+
+<DocLink to="developers/docs/gas/">
+  Gas & Fees
```
Comment (minimalsm) [inline src/content/glossary/index.md:67]:
```suggestion
  Gas and fees
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Hey @zainabb12345, thanks for the PR 😀❤️

Looking good! I've just suggested a few changes based on current site conventions. If you could approve these I'll get this merged or you can let me know if you disagree on any.

**Comment by zainabb12345** [comment]
> Hey @zainabb12345, thanks for the PR grinningheart
> 
> Looking good! I've just suggested a few changes based on current site conventions. If you could approve these I'll get this merged or you can let me know if you disagree on any.

Hey @minimalsm 
Thanks for the suggestions. I've accepted the suggestions, hope I did things right? 
Looking forward to do more contributions :grinning:

**Review by minimalsm — APPROVED** [review]

**Comment by minimalsm** [comment]
Awesome, thanks @zainabb12345. Merging now 🚀

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳! We're currently working on our 2022 POAPs and they'll be released soon.

**Comment by minimalsm** [comment]
@all-contributors please add @zainabb12345 for docs, ideas

**Pull request merged**